tests/behat/features/champ/champ-register-champion.feature

```diff
@@ -0,0 +1,27 @@
+@champ
+@api
+@javascript
+
+Feature: Test registration page for becoming a campus champion
+
+  Scenario: test the proccess of submitting a form to become a campus champion
+    Given I am not logged in
+    When I am on "/form/join-campus-champions"
+    Then I should see "Join Campus Champions"
+    Then I should see "To Join the Campus Champions, we need a letter of collaboration"
+    Then I should see "Letter of Collaboration"
+    When I fill in "username" with "Test"
+    When I fill in "user_first_name" with "Test"
+    When I fill in "user_last_name" with "Test"
+    When I fill in "user_email" with "Test"
+    When I check "champion_user_type"
```

Comment:
@laheara143  -- looks like this `When I check "champion_user_type"` is failing in the test